appcircle_browserstack_espresso/1.0.2/component.yaml

```diff
@@ -0,0 +1,64 @@
+platform: Android
+buildPlatform:
+displayName: BrowserStack App Automate - Espresso
+description: "Run your Espresso tests on BrowserStack App Automate"
+docUrl: "https://docs.appcircle.io/workflows/android-specific-workflow-steps/#browserstack-app-automate---espresso"
+requiredComponents:
+precedingComponents: "appcircle_android_build_ui_test,appcircle_android_build,appcircle_android_sign,appcircle_android_flutter_build,appcircle_android_resign,appcircle_bundletool"
```

Comment:
Are only build alan steps necessary for this comp? Because this component only needs $AC_APK_PATH and $AC_TEST_APK_PATH files. And Android Build for UI Testing must be added first.